empirical-data/geomesa/src/main/scala/com/azavea/ingest/geomesa/Ingest.scala

```diff
@@ -83,7 +83,7 @@ object Ingest {
       var fw: FeatureWriter[SimpleFeatureType, SimpleFeature] = null
       try {
         fw = ds.getFeatureWriterAppend(params.featureName, Transaction.AUTO_COMMIT)
-        featureIter.toStream.foreach({ feature: SimpleFeature =>
+        featureIter.buffer.foreach({ feature: SimpleFeature =>
```

Comment:
Why this change? Also it's `buffered`, not `buffer`. Fixed, but wondering why we needed this.